### PR TITLE
Feature/audio inception model

### PR DESCRIPTION
**Pull request by michsak**

### CODE REVIEW

File: models/audio/train_inception.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    conv_base.trainable = False
+    return model
+
+
+def process(image, label):
+    image = tf.cast(image / 255., tf.float32)
+    return image, label
+
+
+def train_val_split(path, batch_size, img_shape):
```
Comment (Johny-Code) [inline models/audio/train_inception.py]: Can you define this function as tool for testing all models? It should be similar for any type of NN.

File: models/audio/train_inception.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        validation_split=0.2,
+        seed=123,
+        subset='both',
+        color_mode='rgb',
+        image_size=img_shape,
+        batch_size=batch_size)
+    return train_ds.map(process), validation_ds.map(process)
+
+
+def plot_acc_loss(history):
```
Comment (Johny-Code) [inline models/audio/train_inception.py]: The same comment. Define it as tool for all models.

**Review by Johny-Code — CHANGES_REQUESTED** [review]

File: models/audio/train_inception.py (hunk 1 above)
Comment (michsak) [reply models/audio/train_inception.py]: Good idea

File: models/audio/train_inception.py (hunk 2 above)
Comment (michsak) [reply models/audio/train_inception.py]: Yop

**Pull request merged**